recipes/ffmpeg/bld.bat

```diff
@@ -0,0 +1,3 @@
+:: http://ffmpeg.zeranoe.com/blog/?p=300
+
+bash "%RECIPE_DIR%\mingw-w64-build-3.6.0.sh"
```

Comment:
Afraid not. To confirm, my understanding from this morning that this script does nothing with ffmpeg and is entirely to get a suitable mingw toolchain (the comment in the script is along the lines of ```This script will build MinGW-w64 for Win32 and Win64.```). Its been a long day and I'm struggling to see exactly what problem the script solves - sorry for being so slow :snail: